src/CAN/CANMotor.h

```diff
@@ -136,6 +136,8 @@ void setMotorPower(deviceserial_t serial, int16_t power);
  */
 void setMotorPIDTarget(deviceserial_t serial, int32_t target);
 
+void setMotorPIDMaxPower(deviceserial_t serial, uint16_t maxPower);
```

Comment:
This method restricts the maximum power that the motor is allowed to apply when running PID. Add documentation! Also, maybe add an overload with a double parameter for power in the range `[0,1]`?